universal-application-tool-0.0.1/app/views/style/BaseStyles.java

```diff
@@ -2,6 +2,22 @@
 
 public final class BaseStyles {
 
+  public static final String ADMIN_NAV_BAR =
```

Comment:
Should we put this in a file called `AdminStyles.java` similar to the above file? Or if it's used in both views, can rename it `NAV_BAR`.

Reply:
Yeah, still kind of waiting for things to be more concrete before trying to really factor out styles.